### DISCUSSION TITLE
osu!lazer 32bit

### DISCUSSION THREAD

**Discussion by zzahkaboom24** [ppy/osu#13801] [Q&A] [answered]
I know this question was asked a lot of times and that this will only happen if its not stopping anything in the development, i also know that it was said thats its gonna be done eventually. I am just asking, according to the roadmap a new installer will be written. Will this also include 32bit as well? I am on 64bit but my friend is on 32bit and thats why i am asking, if thats not okay to ask ignore this please, thanks

**Comment by bdach** [comment, marked as answer]
There had been an effort to add x86/32-bit support previously, but it lacked testing (https://github.com/ppy/osu-deploy/pull/41).

Whether 32-bit will be supported depends to some degree on how painful that support will be to maintain and how many workarounds (hopefully none) would have to be applied for it to happen.

**Reply by zzahkaboom24** [reply]
Okay. Yea, ive been following this and saw the effort by peppy asking on twitter longer ago, thanks for answering tho and will forward this to my friend.
